src/css/modules/_panel.scss

```diff
@@ -211,6 +211,55 @@
   }
 }
 
+.panel__simplepage {
```

Comment:
Marco,

It seems like the changes do work, correct? We appreciate the suggestions and if you would like to make the fixes yourself you’re welcome to do that, but otherwise we’d like to merge and deploy to Staging.

Thanks,
Kelley

From: Marco Segreto [mailto:notifications@github.com]
Sent: Wednesday, August 26, 2015 2:07 PM
To: 18F/letgirlslearn
Subject: Re: [letgirlslearn] Subscribe button and Logo Guidance Page (#12)


In src/css/modules/_panel.scss<https://github.com/18F/letgirlslearn/pull/12#discussion_r38014428>:

> @@ -211,6 +211,55 @@

>    }

>  }

>

> +.panel__simplepage {

It's hard for me to see what's being done here because I can't see the design, but it does seem like a lot of overriding. I'd probably split some of this logic out into separate components. For instance, if you need a 45em centered content area, then that could be a .panel__small component that you apply to the p tag. In general, this many tag rules under one class seems a little strange.

—
Reply to this email directly or view it on GitHub<https://github.com/18F/letgirlslearn/pull/12/files#r38014428>.